### DISCUSSION TITLE
[Deliver tool] Add check of  the "metadata_path" option to the Deliver tool setup checks

### DISCUSSION THREAD

**Discussion by alexander-smolyakov** [fastlane/fastlane#17567] [Ideas]
Hello,

As I can see the Deliver tool checks several options and also check the existence of a metadata folder before outputting the massage about the setup of the Deliver tool.

https://github.com/fastlane/fastlane/blob/master/deliver/lib/deliver/commands_generator.rb#L68-L78

Could you please add a check of the `metadata_path` option as well?

Since we can specify a custom path to the metadata folder through CLI I assume it makes sense to also check this option before displaying the message: `No deliver configuration found in the current directory. Do you want to setup deliver?`.

Also one more question: Is it possible to somehow skip the setup message when `Fastlane` launched in non-interacted mode?

Thanks!

**Comment by SteveKennedy** [comment]
This still seems to be an issue 2 years later.

It's strictly related to https://github.com/fastlane/fastlane/issues/18321, that got closed with no answer.

If I pass in a custom metadata_path, fastlane wont look in that directory for Deliveryfile.
